.github/ISSUE_TEMPLATE/save_issue.yml

```diff
@@ -0,0 +1,22 @@
+name: Saving Issue
```

Comment:
Can we change the link in the error popup to point directly to this issue template somehow? Currently it automatically pastes the error into the issue body, but it likely needs tweaking to fill the appropriate text field in this new format.

https://github.com/naturalcrit/homebrewery/blob/d20bb15204f563e850605905a0d6cfe251a43d04/client/homebrew/pages/editPage/editPage.jsx#L325

Reply:
I can look into it.   But, with this PR there are two types of issue templates, so it'd have to determine which type of issue is being reported.  That might not be *that* difficult, but it's more of a *thing*.

Reply:
If it's coming from that line in the code we know it's a saving issue, if that helps.

Reply:
ah okay, I didn't look at the greater context of that line.  Yes, that would make it much easier.  Will look at tomorrow*

\**not guaranteed to be tomorrow*

Reply:
This comment shows how you can format a link to auto-fill parts of the issue form: https://github.com/community/community/discussions/15477#discussioncomment-2793106

Reply:
https://stackoverflow.com/questions/34146618/pre-populate-the-github-new-issue-form-using-the-querystring